tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
For faster builds you may want to enable the `incremental` config option (https://www.typescriptlang.org/v2/en/tsconfig#incremental).